Dominic,<div><br></div><div>this is a ROMIO error message, and ROMIO is from MPICH project.</div><div>at first, I recommend you try the same test with the latest mpich, in order to check</div><div>whether the bug is indeed from romio, and has been fixed in the latest release.</div><div>(ompi is a few version behind the latest romio)</div><div><br></div><div>would you be able to post a trimmed version of your application that evidences the test ?</div><div>that will be helpful to understand what is going on.</div><div><br></div><div>you might also want to give a try to</div><div>mpirun --mca io ompio ...</div><div>and see whether this helps.Â </div><div>that being said, I think ompio is not considered as production ready on the v1.10 series of ompi</div><div><br></div><div>Cheers,</div><div><br></div><div>Gilles<br><div><br>On Tuesday, March 15, 2016, Dominic Kedelty &lt;<a href="mailto:dkedelty@asu.edu">dkedelty@asu.edu</a>&gt; wrote:<br><blockquote class="gmail_quote" style="margin:0 0 0 .8ex;border-left:1px #ccc solid;padding-left:1ex"><div dir="ltr"><div>I am getting the following error using openmpi and I am wondering if anyone would have clue as to why it is happening. It is an error coming from openmpi.<br><br>Error in ADIOI_Calc_aggregator(): rank_index(40) &gt;= fd-&gt;hints-&gt;cb_nodes (40) fd_size=213909504 off=8617247540<br>Error in ADIOI_Calc_aggregator(): rank_index(40) &gt;= fd-&gt;hints-&gt;cb_nodes (40) fd_size=213909504 off=8617247540<br>application called MPI_Abort(MPI_COMM_WORLD, 1) - process 157<br>application called MPI_Abort(MPI_COMM_WORLD, 1) - process 477<br><br></div><div>Any help would be appreciated. Thanks.<br></div></div>
</blockquote></div></div>

